Базовая стилизация всех страниц

Nemchenco96 opened the pull request:
Картинки с макета выгрузил, прогнал через редакторы для уменьшения занимаемой памяти, подключил. размеры и описание сделал.
Не стал добавлять мелкие элементы для списков и звезды рейтинга для отелей, как я понял, это при подключении стилей потом лучше сделать?

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2065035/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/2065035-sedona-35/3/

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2065035/tasks/8)

Nemchenco96 committed:
Добавление графика

Картинки с макета выгрузил, прогнал через редакторы для уменьшения занимаемой памяти, подключил. размеры и описание сделал.

Dima-Dim commented:
> Не стал добавлять мелкие элементы для списков и звезды рейтинга для отелей, как я понял, это при подключении стилей потом лучше сделать?

Да, декоративные элементы будут по программе позже.

Dima-Dim commented on catalog.html:
```diff
@@ -8,7 +8,7 @@
     <header class="page-header">
       <a class="header-logo" href="index.html">
-      <img class="page-header-logo" alt="Логотип Sedona" src="">
+      <img class="logo" src="images/logo.svg" width="140" height="70,5" alt="Логотип Sedona"
```
Нет отступа у вложенного тега.

Dima-Dim commented on catalog.html:
```diff
@@ -28,12 +28,12 @@
           <li class="navigation-item">
             <a class="navigation-link" href="#">
-            <span class="visually-hidden"> Поиск </span>
+              <span class="visually-hidden"><img src="images/search.svg" width="20" height="22" alt="Поиск"></span>
```
Если тег `img` обернуть в тега с классом `visually-hidden`, то изображения не отобразится в том месте, где оно ожидается. Класс `visually-hidden` используется для текста, который нужно доступно скрыть.

Dima-Dim reviewed (changes requested):
images/HTML_Academy_logo.svg и подобные - https://up.htmlacademy.ru/htmlcss/35/criteries#PROJ-03
[hearts.svg](https://github.com/htmlacademy-htmlcss/2065035-sedona-35/pull/3/files#diff-8c8fa7d19be79e048a283878ef3b07fd3d8ea411e6f23a5b80dcfb053479b470) - как быть если в избранном окажется больше или меньше единиц?

[main_logo1.jpg](https://github.com/htmlacademy-htmlcss/2065035-sedona-35/pull/3/files#diff-4a2af44070813fab0a4d3a364730324d7f679d81b21b06efd2c0d44a2a4bb19c) - что-то как-то очень много всего в этом лого.

Nemchenco96 committed:
Исправление ошибок

1) В названиях файлов указал везде прописные буквы с нижнем подчеркиванием;
2) Лайк убрал, я так понимаю, счетчик лайков будем через js делать, там через стили обрисуем его;
3) Отступы поправил
4) Класс visually-hidden изменил у картинки с лупой на search, у лайка оставил, так как мы еще не дошли до его верстки.

Dima-Dim commented on catalog.html:
```diff
@@ -49,10 +49,13 @@
         <ul class="breadcrumbs">
           <li class="breadcrumbs-item">
-          <a class="breadcrumbs-link" href="index.html"><img src="" alt="На главную"></a>
+            <a class="breadcrumbs-link" href="index.html"><img src="images/home_page.svg" weight="12" height="12" alt="На главную"></a>
```
Здесь тоже декоративное изображение, которые не должно быть в разметке.

Dima-Dim commented on catalog.html:
```diff
@@ -49,10 +49,13 @@
           </li>
           <li class="breadcrumbs-item">
-          <a class="breadcrumbs-link">Гостиницы</a>
+            <img src="images/should.svg" weight="6,39" height="9,94" alt="">
```
А это что за элемент списка появился?

Dima-Dim commented on catalog.html:
```diff
@@ -130,19 +133,19 @@ <h3>Найдено гостиниц: 38</h3>
 
           <a href="#" class="button button-light">
-          <span class="visually-hidden">Показать карточки</span>
+          <span class="visually-hidden"><img src="images/list1.svg" width="48" height="48" alt="Показать рядами"></span>
```
Изображение внутри тега `visually-hidden` не убед видно? В чём смысл? Ну и это тоже декоративное изображение которое не нужно в разметке.

Dima-Dim reviewed (changes requested):
Не все замечания из предыдущей проверки исправлены и добавлены новые ошибки.
images/hearts.svg - всё ещё включает в себя цифру.
images/HTML_Academy_logo.svg - https://up.htmlacademy.ru/htmlcss/35/criteries#PROJ-03

Nemchenco96 committed:
Исправление-2

1) Название изображений сделал строчными буквами и, где составное название через дефис.
2) Где нужно, заменил класс visually-hidden
3) Сердечко оставил без цифр
4) Декоративные элементы из разметки убрал. Кстати, вопрос по поиску и лайку, а они не декоративные?
5) Удалил лишние картинки из папки

Nemchenco96 committed:
Исправление после консультации

Dima-Dim commented on catalog.html:
```diff
@@ -28,12 +29,12 @@
           <li class="navigation-item">
             <a class="navigation-link" href="#">
-            <span class="visually-hidden"> Поиск </span>
+              <span class="search"> Поиск </span>
```
Почему `visually-hidden` заменён на `search`?

Nemchenco96 replied:
Я так понимаю, класс к декоративному элементу мы будем применять непосредственно к ссылке?

Dima-Dim replied:
Это вам расскажут в разделе про декоративные элементы.

Dima-Dim commented on index.html:
```diff
@@ -43,20 +48,20 @@
+        <h2 class="visually-hidden">8 причин, почему Sedona лучше</h2>
         <ul class="advantages-list">
-          <li class="advantages-item">
+          <li class="advantages-item-1">
```
Про цифры в названиях классов мы с тобой проговорили, надеюсь что ты будешь использовать их только там, где без них действительно никак.

Dima-Dim reviewed (changes requested):
images/House.svg - https://up.htmlacademy.ru/htmlcss/35/criteries#PROJ-03

Nemchenco96 committed:
Попытка 5

Не понимаю, почему House.SVG в пуле написан с большой буквы, при копировании сразу же поменял в папке, ну ладно.
Цифры тоже убрал. Про visually-hidden вернул обратно. Как я понял, для декоративного элемента класс применяем непосредственно к ссылке потом?

Nemchenco96 committed:
Базовая стилизация

Dima-Dim commented on styles/styles.css:
```diff
@@ -0,0 +1,292 @@
+}
+
+body {
+font-family: "PT Sans", sans-serif;
```
Почему нет отступов?

Nemchenco96 replied:
Исправил

Dima-Dim commented on styles/styles.css:
```diff
@@ -0,0 +1,292 @@
+}
+
+.advantages-item-image {
+font-family: "PT Sans", sans-serif;
```
Разве `font-family` не наследуется?

Nemchenco96 replied:
Точно. Шрифт один. Уже исправил
